src/features/auth/Register.tsx

```diff
@@ -27,26 +44,23 @@ function useRegister() {
   const onSubmit = useCallback(
     async (
       name: string,
-      lastname: string,
       email: string,
       password: string,
       confirmPassword: string
     ) => {
       setLoading(true);
       try {
-        if (!name || !lastname || !email || !password || !confirmPassword)
-          return;
+        if (!name || !email || !password || !confirmPassword) return;
```

Comment:
Faltan mensajes de error, el usuario le dara click a registrarse y no va a pasar absolutamente nada, se va a confundir y no sabra por que pasa esto.
```suggestion
        if (!name || !email || !password || !confirmPassword) return;
```
Incluso una mejor aproximacion a los mensajes de error seria poner los inputs en rojos y poner especificamente el error alli, pero de momento bastaria incluso con los mensajes de error.